src/Illuminate/Validation/Rules/Password.php

```diff
@@ -124,7 +124,7 @@ public function __construct($min)
      * If no arguments are passed, the default password rule configuration will be returned.
      *
      * @param  static|callable|null  $callback
-     * @return static|null
+     * @return static|void
```

Comment:
This might be better suited for a separate PR as this is not strongly related to the rest of this PR

Reply:
It's an incorrect docblock in the two classes that are similar to this PR. This PR uses the correct return type and fixes it for the other usages. I think it's probably too small for a standalone PR, but will take it out and make a separate PR if the Laravel team prefers that

Reply:
I usually tiptoe around Taylor and something as small as this or a fly on his office wall can make him reject a PR, so I better don't take any chances.